Firmware/mmu2_protocol_logic.cpp

```diff
@@ -796,7 +793,6 @@ StepStatus ProtocolLogic::Step() {
         break;
     case VersionMismatch:
         LogError(PSTR("Version mismatch"));
-        Stop(); // cannot continue
```

Comment:
**Note:** Removed this call because it's redundant. `Stop()` is called via `StopKeepPowered()` when the `VersionMismatch` error is reported on level higher.